modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartDioNextClientCodegen.java

```diff
@@ -88,11 +92,17 @@ public DartDioNextClientCodegen() {
         modelPackage = "lib.src.model";
 
         supportedLibraries.put(SERIALIZATION_LIBRARY_BUILT_VALUE, "[DEFAULT] built_value");
+        supportedLibraries.put(SERIALIZATION_LIBRARY_JSON_SERIALIZABLE, "json_serializable");
```

Comment:
Do we want to add some beta/experimental tag here? `[BETA] json_serializable`?

Reply:
Probably not a bad idea. Like I've said earlier we've been using this branch in production since August last year, but there are a bunch of OpenAPI features that we don't make use of.